R/pkg/R/client.R

```diff
@@ -60,6 +60,15 @@ generateSparkSubmitArgs <- function(args, sparkHome, jars, sparkSubmitOpts, pack
   combinedArgs
 }
 
+determineLauncher <- function(sparkSubmitBin, combinedArgs, capture = FALSE) {
```

Comment:
Sure. I will. 

BTW, could you inform me if I can disable style checking for some blocks? It seems it fails due to the usages of `shell`.